README.md

```diff
@@ -96,7 +253,7 @@ Docker build process, the dependencies are installed under `/usr/src/node-red`.
 The main sections to modify are
 
     "dependencies": {
-        "node-red": "0.14.x",           <-- set the version of Node-RED here
+        "node-red": "0.18.x",           <-- set the version of Node-RED here
```

Comment:
likewise should match latest available

Reply:
Taking care of this in the new coming PR.